docs/dev/developer-guides/transactions/transactions.md

```diff
@@ -1,61 +1,95 @@
-- **Signature**: A signature indicates who sent the communication. The signature is created when an externally owned account confirms and signs the transaction with its private key.
+## Transaction data on zkSync Era
 
-### Transaction Types
+- `Transaction Hash`: Unique 66 character identifier generated when transaction submitted to L2.  
+- `Commit Tx hash`: Unique 66 character identifier generated when transaction committed to L1.  
+- `Prove Tx hash`: Unique 66 character identifier generated when transaction proven on L1. 
+- `Execute Tx hash`: Unique 66 character identifier generated when transaction executed on L1. 
+- `Status`: One of `Pending`, `Included`, `Verified`, or `Failed`. See [Transaction statuses section](#transaction-statuses) below.
+- `Block`: Block number containing the transaction.  
+- `Batch`: Batch number containing the transaction. 
+- `From`: The account or smart contract address sending the transaction. 
+- `Tokens Transferred`: Details on all tokens transferred by the transaction. 
+- `Contract address`:  The transaction recipient. 
+- `Input data`: Any additional data used by the transaction for verified contracts.
+- `Fee`: Fee for the tx in ETH and USD value. Click **More Details** to see info on refunds.
+- `Nonce`: Sender nonce.                         
+- `Created`: Timestamp of when the transaction was added to the block.   
 
-- Simple or asset transfers: This refers to the regular tokens transfer in the form of ether from one account to another.
+## Nature of transactions
 
-- Contract deployment transactions: Contract deployment on zkSync is quite different from Ethereum.
-  - Ethereum: Contract deployment occurs when a user sends a transaction to the zero address `(0x000...000)` with the `data` field of the transaction equal to the contract bytecode concatenated with the constructor parameters.
-  - zkSync Era: To deploy a contract on zkSync, a user calls the `create` function of the [ContractDeployer](../system-contracts.md#contractdeployer) and provides the hash of the contract to be published, as well as the constructor arguments. The contract bytecode itself is supplied in the `factory_deps` field of the EIP712 transactions. If the contract is a factory (i.e. it can deploy other contracts), the contracts bytecodes should be included in the `factory_deps`.
-  Read more on [contract deployment](../../building-on-zksync/contracts/contract-development.md).
+There are various different ways of transacting on the Ethereum blockchain:
 
-::: tip
-zkSync Era supports Ethereum's "old" (pre-EIP-2718) transaction types, the EIP-1559 transaction type, and the EIP-712 transactions; which can be used to access zkSync-specific features such as account abstraction. Furthermore, smart contracts can only be deployed with the EIP-712 transaction type.
+- Common transactions: A simple transaction from one account to another such as transferring a token.
```

Comment:
Transferring a token is also a contract interaction (with the corresponding erc20 contract). The only transaction that does not interact with a contract is a native coin (i.e. Ether) transfer.

Generally, I'm not sure what's the benefit in describing all of that. I mean, why do we describe the "ways of transacting on the Ethereum blockchain"?

Reply:
again, this has been live content so i'm always reluctant to remove it as i can only assume people are aware of it ? I can remove all this section..